### ISSUE TITLE
reddit trash removal blocked uBlock

### ISSUE THREAD

**Issue by RicoHeat** [DandelionSprout/adfilt#800]
### Describe the problem below this line as meticulously and detailed as possible (incl. pagelinks if any)

uBlock for Fennec_fdroid is currently not allowing the mentioned list from loading.. Only the title of the list appears but with zero amount 
[0 of 0]

https://raw.githubusercontent.com/DandelionSprout/adfilt/master/RedditTrashRemovalService.txt

### Which adblocker(s) did you use when testing this?

uBlock Origin

### Which filterlists did you use? Failing to tell this will temporarily close the report until it has been told.

All of uBlock filters in addition to all and every additional list required for "medium mode"
Anti-Malware by DandelionSprout and Legitimate URL Shortener.

### Which browser(s) did you use when testing this?

(Other)

### Browser version(s)

Fennec Fdroid latest.

**Comment by DandelionSprout** [comment]
Does it show a yellow warning triangle next to "0 of 0"?

It thankfully doesn't seem to be on uBO's official "No-load" list: https://github.com/uBlockOrigin/uAssets/blob/master/filters/badlists.txt

**Comment by RicoHeat** [comment]
> Does it show a yellow warning triangle next to "0 of 0"?
> 
> It thankfully doesn't seem to be on uBO's official "No-load" list: https://github.com/uBlockOrigin/uAssets/blob/master/filters/badlists.txt

At first I thought the same thing "badlists"
![Screenshot_20230310-131513_Fennec](https://user-images.githubusercontent.com/41478458/224230738-e397b42f-bf22-487f-b3a2-afdc91d91416.jpg)

**Comment by DandelionSprout** [comment]
Can you check if https://gitlab.com/DandelionSprout/adfilt/-/raw/master/RedditTrashRemovalService.txt or https://cdn.jsdelivr.net/gh/DandelionSprout/adfilt@master/RedditTrashRemovalService.txt work any better?

**Comment by DandelionSprout** [comment]
Okay, I found the problem. This'll sound fairly weird, but this is the cause:

A couple years ago, I was pretty angry that Reddit had begun to haphazardly and brutally enforce logins and its app when using a phone browser, such that I designated every single entry in the list as desktop-only out of shock. And even after I found a loophole months later that allowed most subreddits to be shown if they were opened from a Google search, I never removed that designation. I'll fix that now.
